app/javascript/controllers/files_controller.js

```diff
@@ -5,7 +5,20 @@ export default class extends Controller  {
 
   // Triggered when edit-deposit controller sends an error event
   error(e) {
+    if (e.detail === null || e.detail.length == 0) {
```

Comment:
Can you say more about needing both of these checks? (And should the latter be `===` instead of `==` or nah?)

Reply:
seemed nice to be able to tolerate either a `null` error list or an empty`[]` error list as meaning the same thing in this context.  i had initially passed `null`, but then switched over to `[]`, and so you could argue this is cruft from that.  happy to simplify to the the sole actual usage pattern, if you'd rather, don't feel strongly.

Reply:
oh also, yes, possible that second `==` should be a `===`.  does it make a difference for integer comparison?

Reply:
I... am not sure.